resources/idf-python-wheels_diagram_source.txt

```diff
@@ -0,0 +1,79 @@
+@startuml
+
+:build-wheels-platforms.yml;
+start
+
+:Install OS dependencies (os_dependencies directory);
+
+:Build wheels for IDF (build_wheels.py)
+
+- minimum IDF-supported Python version;
+start
+
+:Fetch all ESP-IDF branches;
+
+:Get used ESP-IDF branches (specified from GitHub variables);
+
+:Get constraints files for used ESP-IDF branches;
+
+:                      Assemble requirements
+- create requirements.txt
+- transform yaml to requirement from exclude_list.yaml
+- exclude defined requirements from requirements.txt
+- create additional requirements from include_list.yaml;
+
+:         Build additional requirements
+- from include_list.yaml
+- print requirements to be built wheels for
+- print statistics;
+if (1 or more wheel(s) build failed) then (yes)
+:Raise Error;
```

Comment:
I don't know if you can influence this but it would be more natural if the successful branch could continue and failure would go to the right. This applies to the second branching as well.